ner_v2/detectors/textual/queries.py

```diff
@@ -199,7 +199,8 @@ def _parse_multi_entity_es_results(results_list):
         for results in results_list:
             entity_dict = {}
             entity_variants_to_values_dict = {}
-
+            if 'hits' not in results:
+                raise KeyError('No hits from ES')
```

Comment:
raising KeyError is fine, but we are missing out on logging the elastic search queries that are generating these errors

Reply:
got it